python/tests/expressions/test_expressions_base.py

```diff
@@ -175,6 +138,50 @@ def test_strs(op, string):
     assert str(op) == string
 
 
+@pytest.mark.parametrize(
+    "a,  schema, case_sensitive, success",
+    [
+        (
+            base.In(base.UnboundReference("foo"), [literal("hello"), literal("world")]),
+            "table_schema_simple",
+            True,
```

Comment:
It's hard to read test cases that are parameterized like this, with boolean literals. Can you separate these into different test methods? I think that it makes more sense to split out `success` since nearly the entire test code is different depending on its value. And it makes sense to split out case sensitive as well since it is easier to follow if you pass it directly to `bind`"

```
    expr.bind(schema, case_sensitive=True).term.field == ...
```